### PR TITLE
Feature/sim1092

### PR DESCRIPTION
**Pull request by yalsayyad**

### CODE REVIEW

File: python/lsst/sims/catUtils/baseCatalogModels/GalaxyModels.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import warnings
 import numpy
 import os
+from .BaseCatalogModels import BaseCatalogObj
 from lsst.sims.catalogs.generation.db import ChunkIterator, CatalogDBObject, ObservationMetaData
+from lsst.daf.persistence import DbAuth
```
Comment (danielsf) [inline python/lsst/sims/catUtils/baseCatalogModels/GalaxyModels.py]: Does this import of DbAuth actually get used?

**Comment by danielsf** [comment]
Just to make sure I'm clear on the overall scheme:

config/db.py tells the CatalogDBObjects which database they should be connecting to

the user's ./lsst/db-auth.py file tells the CatalogDBobjects how to connect to that database (i.e. it actually carries around the passwords and usernames)

true?

**Comment by danielsf** [comment]
Is there supposed to be a policy/db.paf file in this repository?  The unit tests all fail for me with

lsst.pex.exceptions.wrappers.IoError: 
  File "src/PolicyFile.cc", line 237, in virtual void lsst::pex::policy::PolicyFile::load(lsst::pex::policy::Policy &) const
    failure opening Policy file: /Users/danielsf/physics/lsst_150412/Development/sims_catUtils/policy/db.paf {0}
lsst::pex::exceptions::IoError: 'failure opening Policy file: /Users/danielsf/physics/lsst_150412/Development/sims_catUtils/policy/db.paf'

File: python/lsst/sims/catUtils/baseCatalogModels/GalaxyModels.py (hunk 1 above)
Comment (yalsayyad) [inline python/lsst/sims/catUtils/baseCatalogModels/GalaxyModels.py]: Nope. Good catch.

**Comment by yalsayyad** [comment]
Your understanding of the overall scheme is correct. the .lsst/db-auth.paf contains the usernames and passwords for *all* your central databases (e.g. mine has now has my lsst10 and fatboy usernames and passwords) for all packages in which daf.persistence.DbAuth is used. 
The sims_catUtils/config/db.py contains the default database driver/host/port/database that Stars/Ssm/GalaxyModels will query. 

Regarding the pex_policy error you get: There is not supposed to be a policy/db.paf file. After Russell mentioned that pex_policy is being retired in favor of pex_config, I moved the default database specification to config/db.py. Will you post the traceback all the way to the python in the unit test?

**Comment by danielsf** [comment]
tests/testAllObjects.py

Traceback (most recent call last):
  File "tests/testAllObjects.py", line 8, in <module>
    import lsst.sims.catUtils.baseCatalogModels as bcm
  File "/Users/danielsf/physics/lsst_150412/Development/sims_catUtils/python/lsst/sims/catUtils/baseCatalogModels/__init__.py", line 5, in <module>
    from .OpSim3_61DBObject import *
  File "/Users/danielsf/physics/lsst_150412/Development/sims_catUtils/python/lsst/sims/catUtils/baseCatalogModels/OpSim3_61DBObject.py", line 11, in <module>
    class OpSim3_61DBObject(DBObject):
  File "/Users/danielsf/physics/lsst_150412/Development/sims_catUtils/python/lsst/sims/catUtils/baseCatalogModels/OpSim3_61DBObject.py", line 17, in OpSim3_61DBObject
    policy = pexPolicy.Policy(policyFile)
  File "/Users/danielsf/physics/lsst_150412/DarwinX86/pex_policy/10.0-1-g0f53050+18/python/lsst/pex/policy/policyLib.py", line 803, in __init__
    this = _policyLib.new_Policy(*args)
lsst.pex.exceptions.wrappers.IoError: 
  File "src/PolicyFile.cc", line 237, in virtual void lsst::pex::policy::PolicyFile::load(lsst::pex::policy::Policy &) const
    failure opening Policy file: /Users/danielsf/physics/lsst_150412/Development/sims_catUtils/policy/db.paf {0}
lsst::pex::exceptions::IoError: 'failure opening Policy file: /Users/danielsf/physics/lsst_150412/Development/sims_catUtils/policy/db.paf'

**Pull request closed without merging**